src/DataMapper/PropertyMapper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the RichModelFormsBundle package.
+ *
+ * (c) Christian Flothmann <christian.flothmann@sensiolabs.de>
+ * (c) Christopher Hertel <christopher.hertel@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace SensioLabs\RichModelForms\DataMapper;
+
+/**
+ * @author Christian Flothmann <christian.flothmann@sensiolabs.de>
+ */
+interface PropertyMapper
+{
+    public function readPropertyValue($model);
```

Comment:
would be great to use `$data` to be consistent with `DataMapperInterface` and `FormFactory`